src/Runner.Common/JobServerQueue.cs

```diff
@@ -230,21 +232,31 @@ public void QueueFileUpload(Guid timelineId, Guid timelineRecordId, string type,
             _fileUploadQueue.Enqueue(newFile);
         }
 
-        public void QueueSummaryUpload(Guid stepRecordId, string name, string path, bool deleteSource)
+        public void QueueResultsUpload(Guid recordId, string name, string path, string type, bool deleteSource, bool finalize, bool firstBlock, long totalLines)
```

Comment:
`recordId` or `timelineRecordId`, i think the name better matches what you have in the interface.

Reply:
same for `long totalLines` => `long totalLines = 0`

Reply:
Default parameters are a source of bugs because it's possible the default values on the interface and the implementation are different.  Then depending on how the type is initiated, there could be different behaviors.  I believe the best practice is to only have the value defined at the interface level?  This way if someone mistakenly instantiate the concrete type, then they must explicitly pass in the value for this parameter.  

I will update the `recordId` name.